### PR TITLE
MAINT: Add eirini image annotation

### PR DESCRIPTION
**Pull request by paulcwarren**
MAINT: Annotate eirini images
    
    [#174240799](https://www.pivotaltracker.com/story/show/174240799)
    
    Co-Authored-by: Paul Warren <pwarren@pivotal.io>
    Co-Authored-by: Carlos Iriarte <ciriarte@pivotal.io>

> Thanks for contributing to cf-for-k8s!
>
> We've designed this PR template to speed up the PR review and merge process - please use it.

> _Please describe the change here._ 

---

- Make sure this PR is based off the `develop` branch
- If you're adding/removing/changing any values in `config/values.yml`, please review [this "cf-for-k8s values for contributors" doc](https://docs.google.com/document/d/1Y3jAx48TCGIQdzOFmzqp_R_0WT8Hfjx9oyqs2Tk0Otw/edit#) for up-to-date principles and guidelines for contributing values changes.
- Checkout the [contributing guidelines](https://github.com/cloudfoundry/cf-for-k8s/blob/develop/community/CONTRIBUTING.md)
- Include any links to related PRs, issues, stories, slack discussions, etc... that will help establish context.
- Is there anything else of note that the reviewers should know about this change?

**Acceptance Steps**

_please replace this with a series of instructions (eg kubectl or cf cli commands) for how our Product Manager can verify that your changes were properly integrated_

_Tag your pair, your PM, and/or team_

> _It's helpful to tag a few other folks on your team or your team alias in case we need to follow up later._

### CODE REVIEW

File: config/eirini/_ytt_lib/eirini/rendered.yml [hunk 1]
```diff
@@ -1008,8 +1008,10 @@ kind: Deployment
 metadata:
   annotations:
     kbld.k14s.io/images: |
-      - Metas: null
-        URL: index.docker.io/eirini/opi@sha256:c7c60d73c89db5c1a74019b86db0fb93686ed5572984411f1d4a027681aec457
+      - Metas:
+        - Type: preresolved
+          URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-opi@sha256:1f08f8ca4e6981034043612159f7268e43e84a9d5d000c6c88f802c54fc5d265
+        URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-opi@sha256:1f08f8ca4e6981034043612159f7268e43e84a9d5d000c6c88f802c54fc5d265
```
Comment (Birdrock) [inline config/eirini/_ytt_lib/eirini/rendered.yml:1014]: Looks like duplicates here?

File: config/eirini/_ytt_lib/eirini/rendered.yml [hunk 2]
```diff
@@ -1135,8 +1137,10 @@ kind: Deployment
 metadata:
   annotations:
     kbld.k14s.io/images: |
-      - Metas: null
-        URL: index.docker.io/eirini/event-reporter@sha256:62a30205593b22dfb954dfe803c0e01866e4ac08717ec3089c4c32aec2f00457
+      - Metas:
+        - Type: preresolved
+          URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-event-reporter@sha256:b9e2218b95aa2557b5d85a1c31c6a0b4a60156a78afaea56ceac1bbe7787996a
+        URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-event-reporter@sha256:b9e2218b95aa2557b5d85a1c31c6a0b4a60156a78afaea56ceac1bbe7787996a
```
Comment (Birdrock) [inline config/eirini/_ytt_lib/eirini/rendered.yml:1143]: Looks like duplicate image references.

File: config/eirini/_ytt_lib/eirini/rendered.yml [hunk 3]
```diff
@@ -1193,8 +1197,10 @@ kind: Deployment
 metadata:
   annotations:
     kbld.k14s.io/images: |
-      - Metas: null
-        URL: index.docker.io/eirini/task-reporter@sha256:23e081f0875c40f59526aa2f3bc77cd25c26776d840b2e6e49c4b0f2cbf5a04a
+      - Metas:
+        - Type: preresolved
+          URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-task-reporter@sha256:142847a256db48e0590a23b74b78d6e2086dc8913019e5b6b8050d4a9255f37f
+        URL: index.docker.io/relintdockerhubpushbot/cf-for-k8s-eirini-task-reporter@sha256:142847a256db48e0590a23b74b78d6e2086dc8913019e5b6b8050d4a9255f37f
```
Comment (Birdrock) [inline config/eirini/_ytt_lib/eirini/rendered.yml:1203]: Same comment as above.

File: config/eirini/_ytt_lib/eirini/rendered.yml (hunk 1 above)
Comment (paulcwarren) [reply config/eirini/_ytt_lib/eirini/rendered.yml:1014]: Hmmm...as I understand it these annotations are generated by kbld so I wonder whats up with that?  Will investigate

**Comment by evanfarrar** [comment]
It looks like all PR pipelines are failing so we are going to hold off on investigating, but let me know if this becomes a failure that we should investigate

File: config/eirini/_ytt_lib/eirini/rendered.yml (hunk 1 above)
Comment (paulcwarren) [reply config/eirini/_ytt_lib/eirini/rendered.yml:1014]: https://kubernetes.slack.com/archives/CH8KCCKA5/p1597125371477200

Comment (paulcwarren) [reply config/eirini/_ytt_lib/eirini/rendered.yml:1014]: According to Dmitriy this is not a duplicate it is intended behavior.  See slack thread.

**Pull request merged**